### PR TITLE
Microsoft Entra ID Rebrand

### PR DESCRIPTION
**Pull request by DidunAyodeji**
Updating all Azure AD terms to the Microsoft Entra ID terms in all md files.

## Does this introduce a breaking change?
```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
The updated changes to Microsoft Entra ID terminology make sense.

## Other Information
Here's a link to the Rebrand guidelines. Note that the script doesn't have all the entries in the glossary that mine has. Let me know if you'd like to see my version of the script. https://review.learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad?branch=main

### CODE REVIEW

File: samples/msal-browser-samples/ChromiumExtensionSample/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ This folder contains a sample Chromium extensions demonstrating how to integrate
...
+Chromium extensions are unable to perform certain types of navigation, so applications should leverage the [`chrome.identity.launchWebAuthFlow`](https://developer.chrome.com/apps/identity#method-launchWebAuthFlow) API to perform interactive auth requests. This API takes a url to navigate to, and a callback that will be invoked once the auth flow is completed (which is signaled by ME-ID redirecting back to the `chromiumapp.com` url mentioned earlier). Chromium extensions using MSAL Browser can build working auth flows by composing the MSAL `loginRedirect/acquireTokenRedirect` and `handleRedirectPromise` APIs to generate a url and handle the response:
```
Comment (hectormmg) [inline samples/msal-browser-samples/ChromiumExtensionSample/README.md:23]: Should we go with Microsoft Entra ID here? In isolation some people may not know what this means.

File: samples/msal-browser-samples/TestingSample/Readme.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ## About this sample
 
 This sample demonstrates how you can run e2e tests against an application that uses msal-browser to obtain tokens and sign users in.
-Using the [ROPC flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) in [msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node) you can pre-populate local or session storage with tokens without requiring your test to navigate through the AAD sign-in pages. This allows you to test your application with a real user and real tokens without testing 3rd party sites.
+Using the [ROPC flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) in [msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-node) you can pre-populate local or session storage with tokens without requiring your test to navigate through the ME-ID sign-in pages. This allows you to test your application with a real user and real tokens without testing 3rd party sites.
```
Comment (hectormmg) [inline samples/msal-browser-samples/TestingSample/Readme.md:8]: Same regarding ME-ID/Microsoft Entra ID

File: samples/msal-node-samples/auth-code-cli-brokered-app/README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ const config = {
 
 On the Authenticaion Page of your app, add the following:
 
-- In the **Redirect URI** section, select **Public client/native (mobile & desktop)** in the combo-box and enter the following redirect URI: `ms-appx-web://Microsoft.AAD.BrokerPlugin/<your-client-id>`, replacing `<your-client-id>` with the **Application (client) ID** from your app's registration screen.
+- In the **Redirect URI** section, select **Public client/native (mobile & desktop)** in the combo-box and enter the following redirect URI: `ms-appx-web://Microsoft.aad.BrokerPlugin/<your-client-id>`, replacing `<your-client-id>` with the **Application (client) ID** from your app's registration screen.
```
Comment (hectormmg) [inline samples/msal-node-samples/auth-code-cli-brokered-app/README.md:37]: I'm not sure if redirect URIs are case sensitive but is this change necessary?

**Review by hectormmg — APPROVED** [review]

File: samples/msal-browser-samples/ChromiumExtensionSample/README.md (hunk 1 above)
Comment (DidunAyodeji) [reply samples/msal-browser-samples/ChromiumExtensionSample/README.md:23]: You are exactly right! This is something I've been struggling with in the script. We don't want ME-ID there at all except for outlier cases where there's some kind of space constraint. Thank you!

File: samples/msal-browser-samples/TestingSample/Readme.md (hunk 2 above)
Comment (DidunAyodeji) [reply samples/msal-browser-samples/TestingSample/Readme.md:8]: Thanks! I'm going to update all ME-ID in this branch to be Microsoft Entra ID. I will also make this change in all the other branches I'm working on before I create pull requests on them so we don't see ME-ID again.

File: samples/msal-node-samples/auth-code-cli-brokered-app/README.md (hunk 3 above)
Comment (DidunAyodeji) [reply samples/msal-node-samples/auth-code-cli-brokered-app/README.md:37]: It is not. Updated script will prevent this from happening.  Thank you!

**Comment by tnorling** [comment]
Closing as this appears to have been addressed in #6750
